editor-chrome/actions.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External Dependencies
+ */
+import uuid from 'uuid/v4';
+import { partial } from 'lodash';
+
+export function editPost( edits ) {
+	return {
+		type: 'EDIT_POST',
+		edits,
+	};
+}
+
+export function savePost() {
+	return {
+		type: 'REQUEST_POST_UPDATE',
+	};
+}
+
+export function trashPost( postId, postType ) {
+	return {
+		type: 'TRASH_POST',
+		postId,
+		postType,
+	};
+}
+
+/**
+ * Returns an action object used in signalling that the post should autosave.
+ *
+ * @return {Object} Action object
+ */
+export function autosave() {
```

Comment:
Why are these actions of the chrome? Does it mean if I embed an editor I don't get autosave, etc?

Reply:
Yes, this PR decouples the "post" from the "blocks". Meaning the `editor-chrome` module handles all the Post Save, Update, Creation ... and the `editor` module is a generic block editor module used by the editor chrome to fill the post-content (for now, metaboxes, title ... could come later as a block editor too)